packages/catalog-process/src/model/domain/errors.ts

```diff
@@ -271,3 +273,25 @@ export function audienceCannotBeEmpty(
     title: "Audience cannot be empty",
   });
 }
+
+export function invalidAttributeSeed(
+  eserviceId: EServiceId,
+  descriptorId: DescriptorId
+): ApiError<ErrorCodes> {
+  return new ApiError({
+    detail: `Attributes seed is not valid for EService ${eserviceId} and Descriptor ${descriptorId}`,
```

Comment:
since the payload is not visible in the logs, may I ask to use a more specific message?
feel free to create additional errors if needed (for example for [this](https://github.com/pagopa/interop-be-monorepo/pull/1254/files?diff=split&w=1#diff-77a5bad1f3b877e7bfc968a9b423c16649fca358d54286d743fe34fa555564ceR1719) and [this](https://github.com/pagopa/interop-be-monorepo/pull/1254/files?diff=split&w=1#diff-77a5bad1f3b877e7bfc968a9b423c16649fca358d54286d743fe34fa555564ceR1733) cases)